features/newsletter.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Be able to subscribe to newsletter in multiple pages
+  In order to receive newsletter
+  As an anonymous user
+  I need to be able to subscribe to newsletter from Homepage, any article page, ...
+
+  Scenario: In the homepage I should be able to subscribe with all correct informations
```

Comment:
What about a `Scenario Outline` ?

Reply:
I will try with outline, I didn't think to do it with that